extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -302,6 +302,12 @@ static public function onAlternateEdit( $editPage ) {
 
 	static public function onBeforePageHistory( &$article ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)